test/test_art.py

```diff
@@ -68,17 +68,21 @@ def mock_response(self, url, content_type='image/jpeg', file_type=None):
 
 class FetchImageTest(FetchImageHelper, UseThePlugin):
     URL = 'http://example.com/test.jpg'
+    URL_HTTPS = 'https://example.com/test.jpg'
 
     def setUp(self):
         super(FetchImageTest, self).setUp()
         self.dpath = os.path.join(self.temp_dir, b'arttest')
         self.source = fetchart.RemoteArtSource(logger, self.plugin.config)
         self.extra = {'maxwidth': 0}
         self.candidate = fetchart.Candidate(logger, url=self.URL)
+        self.candidate_https = fetchart.Candidate(logger, url=self.URL_HTTPS)
 
     def test_invalid_type_returns_none(self):
         self.mock_response(self.URL, 'image/watercolour')
+        self.mock_response(self.URL_HTTPS, 'image/watercolour')
         self.source.fetch_image(self.candidate, self.extra)
+        self.source.fetch_image(self.candidate_https, self.extra)
```

Comment:
Since this test controls the URL on "both ends," i.e. the request and the response, I'm not sure it needs updating. Was it failing before?